Documentation/stage1-implementors-guide.md

```diff
@@ -0,0 +1,81 @@
+Stage1 ACI implementor's guide
+=============================
+
+Background
+----------
+
+In rocket the stage1 implementation is what creates the execution environment for the contained applications.  Stage1 execution occurs via entrypoints from stage0 on behalf of `rkt run` and `rkt enter`.  These entrypoints are nothing more than executable programs located via Annotations from within the stage1 ACI manifest.
+
+Entrypoints
+-----------
+
+### `rkt run` => "coreos.com/rocket/stage1/run"
+
+After `rkt run` prepares the stage1 ACI and per-app stage2 ACI file-systems beneath "/var/lib/rkt/containers/$uuid" and writes the Container Runtime Manifest to "/var/lib/rkt/containers/$uuid/container", it chdirs to "/var/lib/rkt/containers/$uuid" and executes the program specified by the "coreos.com/rocket/stage1/run" entrypoint within the the stage1 ACI rootfs at "/var/lib/rkt/containers/$uuid/stage1/rootfs".
```

Comment:
General note: while I am personally usually a fan of prose in docs, this seems like a document where it might be more appropriate to enumerate the set of steps involved as a list instead; I think it's easier to reference (and potentially use as a checklist) when people are e.g. actually working on an implementation